plugin.xml

```diff
@@ -28,13 +28,14 @@
 		<engine name="cordova" version=">=3.0.0"/>
 	</engines>
 
-	<dependency id="org.apache.cordova.device"/>
-	<dependency id="org.apache.cordova.globalization"/>
+	<dependency id="cordova-plugin-device"/>
+	<dependency id="cordova-plugin-globalization"/>
 
 	<!-- android -->
 	<platform name="android">
 
-		<dependency id="com.google.playservices" url="https://github.com/MobileChromeApps/google-play-services.git"/>
+		<framework src="com.google.android.gms:play-services:7.5.0" />
```

Comment:
What you really need to import is gcm, like this: https://github.com/jeduan/cordova-plugin-push-notification/blob/master/plugin.xml#L65